lib/v2/screens/app/interactor/viewmodels/app_bloc.dart

```diff
@@ -69,15 +71,10 @@ class AppBloc extends Bloc<AppEvent, AppState> {
 
   void initDynamicLinks() async {
     FirebaseDynamicLinks.instance.onLink(onError: (error) async {
-      print('onLinkError');
-      print(error.message);
```

Comment:
If there is an error in the reception of the link, is it ignored?

Reply:
for now yes. 
I think we should handle this error case too. 

Next pr will do so